install/kubernetes/helm/istio/values.yaml

```diff
@@ -454,3 +454,12 @@ global:
   # This field is set to false by default, so 'helm template ...'
   # will ignore the helm test yaml files when generating the template
   enableHelmTest: false
+
+
+
+
+#
```

Comment:
can you move this to line 91 so its in the vicinity of the other enable flags?

Reply:
Done